style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 60px;
+}
+
+.container-fluid {
+  background-color: #eee;
+}
+
+.header {
+  text-align: center;
+}
+
+#imagetext {
+  background-color: white !important;
+}
+
+.main {
+  max-width: 660px; /* for  below IE7 use `width: 660px;` */
+  margin: 0 auto;
+}
+
+li span {
+  font-weight: bold;
+}
```

Comment:
Please make changes to the span tag by making it bold instead of span. I think it would be better because of the changes that were made to the HTML.